webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/isNodeUsed.scala.txt

```diff
@@ -22,10 +22,7 @@ PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
 PREFIX salsah-gui: <http://www.knora.org/ontology/salsah-gui#>
 
 SELECT DISTINCT ?isUsed
-@* Ensure that inference is not used in this query. *@
-@if(triplestore.startsWith("graphdb")) {
-    FROM <http://www.ontotext.com/explicit>
-}
+
```

Comment:
in all those templates, the `triplestore` parameter should not be used anymore, and therefor should not have to be passed to the function anymore either

Reply:
I'm still not sure if I should do that in this task, because this would also trigger removal of other things related to triplestore-specific code and documentation.

Reply:
I think that removing the triplestore parameter would't be complicated, just more work. One would usually provide the value directly from settings to the template, e.g.,

```
...
triplestore = settings.triplestoreType,
...
```

So basically this line would need to be removed at every call-sight of a template.